SingInTests/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+using DTO.RestRequests;
+using Kernel;
+using Kernel.Enums;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace SingInTests
+{
+    internal class SignInTest
+    {
+        private CreateUserRequest request = new CreateUserRequest
+        {
+            FirstName = "Test",
+            LastName = "Test",
+            Birthday = DateTime.Today.AddYears(-20),
+            Email = "testUser@mail.ru",
+            Password = "password"
+        };
+        private static void Main(string[] args)
+        {
+            var test = new SignInTest();
+            try
+            {
+                test.TryToSingUp();
+            }
+            catch (Exception e)
```

Comment:
Do we have different types of exceptions? Why don't custom ?

Reply:
Why do we need custom exception ?

Reply:
I wasn't sure, try to understand how gui tests works

Reply:
This test simulates entering the necessary data into the login and password and then clicking send